components/alert/alert-toast.js

```diff
@@ -256,21 +337,44 @@ class AlertToast extends LitElement {
 				}
 			}
 			this.setAttribute('role', 'alert');
+			requestAnimationFrame(() => {
+				const height = this._innerContainer.offsetHeight;
+				this.dispatchEvent(new CustomEvent(
+					'd2l-alert-toast-open', {
+						bubbles: true,
+						composed: false,
+						detail: { height }
+					}
+				));
+			});
 		} else {
+			if (!this._innerContainer) return;
```

Comment:
This was needed, otherwise there were issues with the visual diff tests and the `d2l-alert-toast-close` event would get fired.

It seems to me that it makes sense that the close functionality doesn't run on initial load but it's possible something was being relied upon here?

Reply:
Cool. I noticed something similar in expand-collapse-content where it doesn't want to fire the event during the initialization. I think it was tracking `_isFirstUpdate` and using that -- `_innerContainer` gets set during `firstUpdated` so that's kind of accomplishing the same thing.